go.mod

```diff
@@ -94,40 +86,36 @@ require (
 
 replace (
 	github.com/go-check/check => github.com/go-check/check v0.0.0-20180628173108-788fd7840127
-	k8s.io/api => k8s.io/api v0.23.1
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.23.1
-	k8s.io/apimachinery => k8s.io/apimachinery v0.23.1
-	k8s.io/apiserver => k8s.io/apiserver v0.23.1
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.23.1
-	k8s.io/client-go => k8s.io/client-go v0.23.1 // Required by prometheus-operator
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.23.1
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.23.1
-	k8s.io/code-generator => k8s.io/code-generator v0.23.1
-	k8s.io/component-base => k8s.io/component-base v0.23.1
-	k8s.io/component-helpers => k8s.io/component-helpers v0.23.1
-	k8s.io/controller-manager => k8s.io/controller-manager v0.23.1
-	k8s.io/cri-api => k8s.io/cri-api v0.23.1
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.23.1
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.23.1
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.23.1
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.23.1
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.23.1
-	k8s.io/kubectl => k8s.io/kubectl v0.23.1
-	k8s.io/kubelet => k8s.io/kubelet v0.23.1
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.23.1
-	k8s.io/metrics => k8s.io/metrics v0.23.1
-	k8s.io/mount-utils => k8s.io/mount-utils v0.23.1
+	k8s.io/api => k8s.io/api v0.28.3
```

Comment:
[argo-cd ](https://github.com/argoproj/argo-cd/blob/master/go.mod#L302)is still using `v0.26.4` . Are we sure that `v0.28.3` is compatible with older version in argo-cd?